src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -49,6 +60,57 @@ internal bool AutomaticRedirection
                 _automaticRedirection = value;
             }
         }
+        internal bool UseProxy
+        {
+            get
+            {
+                if (_proxyPolicy == CurlProxyUsePolicy.UseDefaultProxy)
+                {
+                    return false;
+                }
+                return _proxyPolicy != CurlProxyUsePolicy.DoNotUseProxy;
```

Comment:
David, I'm confused.  The current code starts with:
```C#
if (_proxyPolicy == CurlProxyUsePolicy.UseDefaultProxy)
{
    return false;
}
```
and you said the current code is correct, but now you're saying it should return true here.

I believe my version is logically equivalent to the current code.  If the current code is incorrect, that's a different story.  And regardless, there's a simpler way of expressing the condition.